NICE.Identity.Authorisation.WebAPI/Services/JobsService.cs

```diff
@@ -90,5 +91,29 @@ public int DeleteJob(int jobId)
                 throw new Exception($"Failed to delete job {jobId.ToString()} - exception: {e.Message}");
             }
         }
+
+        /// <summary>
+        /// Delete all jobs relating to an organisation
+        /// </summary>
+        /// <param name="organisationId"></param>
+        /// <returns></returns>
+        public void DeleteAllJobsForOrganisation(int organisationId)
+        {
+            try
+            {
+                var jobToDelete = _context.Jobs.Where((j => j.OrganisationId == organisationId)).ToList();

```

Comment:
```suggestion
                var jobsToDelete = _context.Jobs.Where((j => j.OrganisationId == organisationId)).ToList();

```

Should we call it `jobsToDelete`?